### DISCUSSION TITLE
Ansible error while configuring mongodb service

### DISCUSSION THREAD

**Discussion by marpana** [learndevopsonline/learndevopsonline#1922] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/171904070/bbdee5a8-31c3-470d-b7a1-bbf3798e06c8)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/171904070/2b934383-1dc8-43e8-bf6a-f7d3646118e2)

Please help!

Thanks,
Arpana

**Comment by chaithanya1812** [comment]
Hi @marpana 
## Post a screen-shot like this. I need to see you ip-address of instance..
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/3ea04c71-2a74-4209-a6f0-2a3144042d87)

**Comment by ArpanaM** [comment]
Hi,
Now its Working Raghu, I just  restarted everything

Thank you so much for you response!
